src/web/client/common/remoteFetchProvider.ts

```diff
@@ -51,7 +51,7 @@ export async function fetchDataFromDataverseAndUpdateVFS(
 
         if (!response.ok) {
             showErrorDialog(localize("microsoft-powerapps-portals.webExtension.fetch.authorization.error", "Authorization Failed. Please run again to authorize it"), localize("microsoft-powerapps-portals.webExtension.fetch.authorization.desc", "Try again"));
-            sendAPIFailureTelemetry(requestUrl, entity, Constants.httpMethod.GET, new Date().getTime() - requestSentAtTime, response.json.toString());
+            sendAPIFailureTelemetry(requestUrl, entity, Constants.httpMethod.GET, new Date().getTime() - requestSentAtTime, JSON.stringify(response));
```

Comment:
Just for your awareness:

JSON.stringify can throw exception in case of circular references.
// "TypeError: cyclic object value"